salesagility/SuiteCRM
# Fixed #2983 - translation of fullcalendar fixed

## Description
changed option lang to locale

## Motivation and Context
Solves translation issues with weekdays etc. in Calendar

## How To Test This
Change the language from en_us to de_DE and check this url
https://[SUGARDOMAIN]/index.php?module=Calendar&action=index&parentTab=Alle
The weekdays should be translated to German: Sonntag, Montag, Dienstag, Mittwoch, Donnerstag, Frreitag, Samstag. Also the literal "Ganztätig" should be translated.
The buttons "Shared Month" and "Shared Weeks" are not fixed by this bugfix.

## Types of changes

### Final checklist
